docs/sources/source/nlp/joint_intent_slot_filling.rst

```diff
@@ -190,7 +190,7 @@ To train a joint intent slot filling model, run ``joint_intent_slot_with_bert.py
 
     .. code-block:: python
 
-        python -m torch.distributed.launch --nproc_per_node=2 joint_intent_slot_with_bert.py \
+        python torch.distributed.launch --nproc_per_node=2 joint_intent_slot_with_bert.py \
```

Comment:
I am fairly certain you need the -m here.

Reply:
There is a bug here https://nvbugswb.nvidia.com/NvBugs5/SWBug.aspx?bugid=2775777&cmtNo= on this parameter. In python 3, it seems that .py should not be there if -m is used.

Reply:
Fixed it. Thanks!